precompile/contracts/warp/predicate_test.go

```diff
@@ -92,6 +96,9 @@ func init() {
 		panic(err)
 	}
 
+	graniteRules = extras.AvalancheRules{IsGranite: true}
```

Comment:
same comment to use the helper for full valid rule set

Reply:
also we should probably set this in var block if possible

Reply:
> also we should probably set this in var block if possible

+1 there's no need to delay its definition